neo/Consensus/ConsensusService.cs

```diff
@@ -423,6 +424,7 @@ private void OnPrepareRequestReceived(ConsensusPayload payload, PrepareRequest m
             context.Block.ConsensusData.Nonce = message.Nonce;
             context.TransactionHashes = message.TransactionHashes;
             context.Transactions = new Dictionary<UInt256, Transaction>();
+            context.SendersFeeMonitor.ClearSenderFee();
```

Comment:
Clear or create a new one? I prefer to create a new instance.

Reply:
I mean to create a new `SendersFeeMonitor`.

Reply:
I agree with you, @erikzhang.